Odata-docs/connectedservice/getting-started.md

```diff
@@ -13,17 +13,23 @@ ms.topic: article
 
 OData Connected Service is a Visual Studio extension that generates strongly-typed C# and Visual Basic client code for a specified OData service. It generates a `DataServiceContext` class to interact with the service and CLR types for each entity type and complex type in the service model.
 
-The extension is available at [Visual Studio Marketplace](https://marketplace.visualstudio.com/items?itemName=laylaliu.ODataConnectedService) and supports Visual Studio 2019 and 2017.
+The extension is available at Visual Studio Marketplace:
+
+- [OData Connected Service 2022+](https://marketplace.visualstudio.com/items?itemName=marketplace.ODataConnectedService2022) for Visual Studio 2022
+- [OData Connected Service](https://marketplace.visualstudio.com/items?itemName=marketplace.ODataConnectedService) for Visual Studio 2019 and 2017
 
 ## Creating a sample client application using OData Connected Service
 
 In this sample we are going to install the extension and use it to create a sample client application for the sample [TripPin](https://www.odata.org/blog/trippin-new-odata-v4-sample-service/) OData service.
 
-Open Visual Studio 2019 and create a new C# .Net Core project and call the project `ODataClientExample` (.Net Framework is also supported).
+Open Visual Studio 2022 and create a new C# .NET project and call the project `ODataClientExample`.
+
+> [!NOTE]
+> .NET Framework is also support, but you would have to manually install Microsoft.OData.Client 7.x, Microsoft.OData.Core 7.x, Microsoft.OData.Edm 7.x, Microsoft.Spatial 7.x since the latest version of OData core libraries do no support .NET Framework.
```

Comment:
```suggestion
> .NET Framework is also supported, but you would have to manually install Microsoft.OData.Client 7.x, Microsoft.OData.Core 7.x, Microsoft.OData.Edm 7.x, Microsoft.Spatial 7.x since the latest version of OData core libraries do no support .NET Framework.
```